src/linter/report.go

```diff
@@ -457,6 +457,12 @@ function performance_test() {}`,
 			Default: true,
 			Comment: `Report linter error.`,
 		},
+
+		{
+			Name:    "bareReturnInNonVoidFunction",
```

Comment:
Just "bareReturn" sounds better. It's generally a good practice to stay in 1-2 words per checker name.